### ISSUE TITLE
The rendering got messy if I use Cica font

### ISSUE THREAD

**Issue by mopp** [akiyosi/goneovim#83]
Hi
Thanks for your mention on Twitter.

I will report the wired thing.

## Environment/versions

- macOS Catalina Version 10.15.1
  - MacBook Pro (15-inch, 2019)
- Neovim
  - NVIM v0.4.3
  - Build type: Release
  - LuaJIT 2.0.5
- gonvim
  - Latest commit 3cf9112
  - with the config below

```toml
[Editor]
# Editor external font-family, font-size, and linespace.
FontFamily = "Cica"
FontSize = 14
Linespace = 0
```

- My .vimrc
  - https://github.com/mopp/dotfiles/blob/ce32d8cb793ed95e5e41bab6b51c6b0d40e8f237/.vimrc

### With my .vimrc
<img width="806" alt="Screen Shot 2019-12-09 at 15 00 40" src="https://user-images.githubusercontent.com/1330045/70411643-2831f280-1a96-11ea-940d-550995a71e52.png">

### Without my .vimrc
<img width="807" alt="Screen Shot 2019-12-09 at 15 02 11" src="https://user-images.githubusercontent.com/1330045/70411664-33851e00-1a96-11ea-9236-a83a523862ae.png">

**Comment by akiyosi** [comment]
Hi
Thank you for the report.

I investigate this problem and think that there may be a problem with using of Cica fonts by Qt's text drawing functions.
I will create a minimal application container that reproduces this problem and report the issue to the Cica repository.

**Comment by akiyosi** [comment]
Hi

The problem that the drawing result of the Cica font is messy seems to be something wrong with the Cica font itself, but I found a way to avoid this problem on the application side.
I will include this fix in v0.4.2.

**Comment by mopp** [comment]
That's great!
Thank you for your work :+1:
